packages/web-components/tests/e2e-storybook/cypress/integration/carousel/carousel.e2e.js

```diff
@@ -0,0 +1,343 @@
+          expect($icon).to.have.attr(
+            'd',
+            'M26.5555476,43.111135 C26.0032708,43.111135 25.5555476,42.6633959 25.5555476,42.1111111 L25.5555476,20.1111111 C25.5541311,19.7531358 25.7441673,19.4217049 26.0538295,19.2421008 C26.3634918,19.0624967 26.745539,19.0621192 27.0555476,19.2411111 L46.0555476,30.2411111 C46.368866,30.4186723 46.5625038,30.7509842 46.5625038,31.1111111 C46.5625038,31.4712381 46.368866,31.8035499 46.0555476,31.9811111 L27.0555476,42.9811111 C26.9031526,43.0674916 26.7307319,43.1123209 26.5555476,43.111135 Z'
+          );
+        });
+
+      cy.takeSnapshots();
+    });
+  },
+  checkVideoDurationText: () => {
+    it('should render the video duration in the footer', () => {
+      cy.get(_selectors.videoFooter)
+        .find('span.bx--card__cta__copy')
+        .then($duration => {
+          expect($duration).not.to.be.empty;
+        });
+    });
+  },
+  checkSameHeight: () => {
+    it('should have headings all the same height', () => {
+      cy.get(_selectors.heading)
+        .wait(1000)
+        .then($heading => {
+          let headingHeightNum = $heading.first().height();
+          let headingHeight = headingHeightNum + 'px';
+
+          expect($heading).to.not.have.css('height', '0px');
+          expect($heading).to.have.css('height', headingHeight);
+        });
+    });
+  },
+  checkClickableCard: () => {
+    it("should check that the footer's pseudo class takes up entire card to be clickable", () => {
+      cy.get(_selectorBase).then($carousel => {
+        if ($carousel.find(_selectors.footer).length > 0) {
+          cy.get(_selectors.footer)
+            .shadow()
+            .find('a.bx--card__footer')
+            .then($link => {
+              const url = $link.prop('href');
+              expect(url).not.to.be.empty;
+
+              const win = $link[0].ownerDocument.defaultView;
+              const after = win.getComputedStyle($link[0], ':after');
+              const positionValue = after.getPropertyValue('position');
+              const insetValue = after.getPropertyValue('inset');
+
+              expect(positionValue).to.eq('absolute');
+              if (Cypress.browser.name !== 'firefox') {
+                expect(insetValue).to.eq('0px');
+              }
+            });
+        }
+
+        if ($carousel.find(_selectors.videoFooter).length > 0) {
+          cy.get(_selectors.videoFooter)
+            .shadow()
+            .find('a.bx--card__footer')
+            .then($link => {
+              const url = $link.prop('href');
+              expect(url).not.to.be.empty;
+
+              const win = $link[0].ownerDocument.defaultView;
+              const after = win.getComputedStyle($link[0], ':after');
+              const positionValue = after.getPropertyValue('position');
+              const insetValue = after.getPropertyValue('inset');
+
+              expect(positionValue).to.eq('absolute');
+              if (Cypress.browser.name !== 'firefox') {
```

Comment:
Why are we excluding firefox here?

Reply:
As mentioned above, this code is a copy of very similar code elsewhere. Looking at the git history, it seems that was added in this PR: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/7807 It seemed prudent to keep it in, though if it's now unneeded I can remove it.

Reply:
I would remove this for now until someone tells you to add it back in.